uavcan/register/384.Access.1.0.uavcan

```diff
@@ -64,19 +64,60 @@
-#                       |       |           | node-ID for the current transport indicate that the node-ID is not set;
-#                       |       |           | if plug-and-play is supported, it will be used by the node to obtain an
-#                       |       |           | automatic node-ID. Invalid values other than 65535 should be avoided for
-#                       |       |           | consistency. The factory-default value should be 65535.
-# ----------------------+-------+-----------+--------------------------------------------------------------------------
+# common functions. These conventions are not mandatory to follow, but implementers are recommended to adhere because
+# they enable enhanced introspection capabilities and simplify device configuration and diagnostics.
+#
+#   REGISTER NAME PATTERN (POSIX ERE)                 TYPE            FLAGS                   RECOMMENDED DEFAULT
+# =====================================================================================================================
+#
+#   uavcan\.node\.id                                  natural16       mutable, persistent     65535 (unset/PnP)
+#
+# Contains the node-ID of the local node. Values above the maximum valid node-ID for the current transport
+# indicate that the node-ID is not set; if plug-and-play is supported, it will be used by the node to obtain an
+# automatic node-ID. Invalid values other than 65535 should be avoided for consistency.
+#
+# ---------------------------------------------------------------------------------------------------------------------
+#
+#   uavcan\.node\.description                         string          mutable, persistent     (empty)
+#
+# Human-readable verbose description of this specific node defined by the integrator (not vendor).
+#
+# ---------------------------------------------------------------------------------------------------------------------
+#
+#   uavcan\.cookie                                    string          mutable, persistent     (empty)
+#
+# Arbitrary data stored by an external autoconfiguration authority (e.g., a mission computer, an auto-configuration
+# utility, etc.). This data is never to be used by the node itself; instead, the node merely provides raw data
+# storage space.
+#
+# ---------------------------------------------------------------------------------------------------------------------
+#
+#   uavcan\.pub\.[a-zA-Z_][a-zA-Z0-9_]{0,34}\.id      natural16       mutable, persistent     65535 (unset, invalid)
+#   uavcan\.sub\.[a-zA-Z_][a-zA-Z0-9_]{0,34}\.id      ...             ...                     ...
+#   uavcan\.cln\.[a-zA-Z_][a-zA-Z0-9_]{0,34}\.id      ...             ...                     ...
+#   uavcan\.srv\.[a-zA-Z_][a-zA-Z0-9_]{0,34}\.id      ...             ...                     ...
+#
+# Publication/subscription/client/server port-ID, respectively. These registers are configured by the system integrator
```

Comment:
We should also add a note that these can be autoconfigurable.

Reply:
Done.